### ISSUE TITLE
[MAS2.1.1] Unable to access side panel controls of Bot Framework Emulator using keyboard.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-Emulator#726]
**Actual Result:**
Side panel controls like 'Bot Explorer', 'Services', 'Settings' etc. are of Bot Framework Emulator are not accessible using keyboard.

**Expected Result:**
All interactive controls present in the Bot Framework Emulator should be accessible using keyboard.

_NOTE:_
Bot Services Section is also not accessible using keyboard.
Bot Settings Button is not accessible using keyboard.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Click on open bot and choose the .bot file that is available in the bot folder.
3. Notice side panel of this application with 'Bot Explorer', 'Services', 'Settings', 'Endpoint' and 'Transcript Explorer' etc.
4. Try to navigate and select all these controls using keyboard.

**User Impact:**
Keyboard dependent users won't be able to use the controls which are not accessible using keyboard in any way.

**Test Environment:**
OS: Windows 10
OS Build: 17134.167
OS Version: 1803
Browser: Edge

**Impacted MAS Link:**
[MAS 2.1.1 – Keyboard (36)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2})

**Attachment:**
[MAS2.1.1_BotFrameworkEmulator_SidePaneControls.zip](https://github.com/Microsoft/BotFramework-Emulator/files/2257101/MAS2.1.1_BotFrameworkEmulator_SidePaneControls.zip)
